src/drivers/boards/aerofc-v1/aerofc_usb.c

```diff
@@ -78,13 +78,6 @@
 
 __EXPORT void stm32_usbinitialize(void)
```

Comment:
[sercon](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/rcS#L41) may be an issue.  It may need an "if then else" to allow it to fail